pyspades/server.py

```diff
@@ -177,6 +179,11 @@ def broadcast_contained(self, contained, unsequenced=False, sender=None,
     # backwards compatability
     send_contained = broadcast_contained
```

Comment:
This should probably be commented out considering that we comment out the `send_chat` below